docs/community/SIG-Governance.md

```diff
@@ -0,0 +1,45 @@
+# OCM SIG Governance Model
+
+This document defines the governance framework for Special Interest Groups (SIGs) in the Open Component Model (OCM) project. It is designed to be lightweight and scalable for a small open source project, while ensuring transparency, consistency, and community alignment.
+
+## 1. Purpose
+
+SIGs are groups of contributors focused on specific technical or community topics within OCM. The governance model ensures clear roles, decision-making, and lifecycle management for all SIGs.
+
+## 2. Roles & Responsibilities
+
+- **Chair**: Facilitates meetings, represents the SIG, ensures process adherence. (At least one Chair is required.)
+- **Tech Lead**: Guides technical direction, reviews contributions, supports the Chair. (At least one Tech Lead is required.)
+- **Members**: Regular contributors with voting rights.
+
+## 3. Decision-Making
+
+- Prefer consensus; if consensus cannot be reached, a simple majority vote among Members decides.
+- Major decisions (scope, leadership changes) require approval by the OCM Technical Steering Committee (TSC).
+- All decisions must be documented in meeting notes or SIG repository. Decisions requiring TSC approval are recorded in the OCM repository and meeting minutes.
+
+## 4. Conflict Resolution
+
+- Attempt resolution within the SIG.
+- If unresolved, escalate to the OCM TSC for mediation and final decision. The TSC will review the issue, facilitate discussion, and record the resolution in the community repository and meeting minutes.
+
+## 5. SIG Lifecycle
+
+Lifecycle management covers creation, operation, and archiving/dissolution of SIGs. For detailed steps and requirements, see the [SIG Lifecycle Guide](./SIG-Lifecycle.md).
+
+- **Creation**: Submit a SIG proposal using the official template ([SIG Submission Template](./SIG-Submission-Template.md)). Requires TSC approval.
```

Comment:
```suggestion
- **Creation**: Submit a SIG proposal using the official template ([SIG Submission Template](./SIG-Submission-Template.md)). Requires TSC approval through Vote.
```

Also make sure that the SIG proposal is entered into the TSC Call Agenda

Reply:
done